### PR TITLE
Added "getMIDIValuesForElement" method

### PR DESCRIPTION
**Pull request by sonovice**
Added a method to get the MIDI Pitch for a Note with the given ID after `renderToMIDI()` has been called.
See #944

### CODE REVIEW

File: emscripten/buildToolkit [hunk 1]
```diff
@@ -177,6 +177,7 @@ $exports .= "'_vrvToolkit_getOptions',";
 $exports .= "'_vrvToolkit_getPageCount',";
 $exports .= "'_vrvToolkit_getPageWithElement',";
 $exports .= "'_vrvToolkit_getTimeForElement',";
+$exports .= "'_vrvToolkit_getMIDIPitchForElement',";
```
Comment (lpugin) [inline emscripten/buildToolkit]: Rename it to `getMIDIValuesForElement` and sort it alphabetically.

File: include/vrv/toolkit.h [hunk 2]
```diff
@@ -195,6 +195,13 @@ class Toolkit {
      */
     int GetTimeForElement(const std::string &xmlId);
 
+    /**
+     * Return the MIDI pitch of the element with the ID (xml:id).
+     * RenderToMidi() must be called prior to using this method.
+     * Returns -1 if no element is found.
+     */
+    int GetMIDIPitchForElement(const std::string &xmlId);
```
Comment (lpugin) [inline include/vrv/toolkit.h]: Change it to GetMIDIValuesForElement and return a string / json object with for now only the pitch. We can then use the same method if one day we want to add other MIDI values

Comment (lpugin) [reply include/vrv/toolkit.h]: Sounds good

Comment (sonovice) [reply include/vrv/toolkit.h]: Done, ready to pull.

**Pull request merged**